src/types/quizzes.d.ts

```diff
@@ -1,11 +1,12 @@
-import { ConstructorClientOptions, NetworkParameters, UserParameters } from '.';
```

Comment:
Not related to this line. Can we update `src/types/tests/quizzes.test-d.ts`? `npm run test:types` is failing ATM